lib/auth/auth_with_roles.go

```diff
@@ -926,9 +925,13 @@ func (a *ServerWithRoles) GetNodes(ctx context.Context, namespace string, opts .
 // ListResources returns a paginated list of resources filtered by user access.
 func (a *ServerWithRoles) ListResources(ctx context.Context, req proto.ListResourcesRequest) (*types.ListResourcesResponse, error) {
 	if req.UseSearchAsRoles {
+		clusterName, err := a.authServer.GetDomainName()
```

Comment:
Same comment here as above. I'm not sure when `GetClusterName()` should be preferred over `GetDomainName()`, but the name of the variable should probably match the method call.